tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -93,8 +93,8 @@ class TargetedSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       |  u.<<copy>>(<<age>> = 43)
       |}
     """.trim.stripMargin, { (db, copy, age) =>
-      assert(copy === Symbol("_root_.e.User#copy(Ljava/lang/String;I)Le/User;."))
-      assert(age === Symbol("_root_.e.User#copy(Ljava/lang/String;I)Le/User;.(age)"))
+      assert(copy === Symbol("_root_.e.User#copy(String,Int)."))
```

Comment:
Can we add a targeted test like this here to highlight the disambiguation like `List+1`/`List+2`? It's covered in the expect tests but I think it deserves highlighting in the targeted suite as @MasseGuillaume pointed out.

Reply:
Done.